.github/workflows/production-suite.yml

```diff
@@ -209,13 +209,12 @@ jobs:
         include:
           # Docker images are built from 'tools/ci/Dockerfile'; the comments at
           # the top explain how to build and upload these images.
-          - docker_image: zulip/ci:buster-3.4
-            name: 3.4 Version Upgrade
-            os: buster
-
```

Comment:
Is there a strong reason to stop testing upgrades from 3.4? We do share supported platforms with it, and I think we stopped at 2 major releases just because we ran out of steam.  I guess the main reason would be the extra 7 minutes or so of CI time.

Reply:
Just test time, yeah.  We could maybe only run them on `main` and not on PRs, as a middle ground.

Reply:
OK, let's try leaving it in; there's also an argument that we should cut 4.x rather than 3.x if we're going to cut something.